sdks/python/apache_beam/io/gcp/bigtableio.py

```diff
@@ -268,3 +272,95 @@ def process(self, row):
             partial_row.cells[fam_name][col_qualifier_bytes].append(
                 Cell(value, timestamp_micros))
       yield partial_row
+
+
+class WriteToBigtableXlang(beam.PTransform):
```

Comment:
There's an existing native `WriteToBigTable` transform: https://github.com/apache/beam/blob/e07c461cd45e3942fa48ab4499ec4ee3317dbcff/sdks/python/apache_beam/io/gcp/bigtableio.py#L170-L177

Figured I should make a distinction between the two, but maybe there's a better way than adding Xlang to the end?

Reply:
Ah, that's confusing. How about following ?

Introduce a keyword argument "use_cross_language" to the constructor of WriteToBigTable. If this is set to True, use your implementation underneath (in the composite) instead of the existing implementation. For now leave the default value of the property to "False" to that any existing users do not run into surprises.

In the future, we should change the default value of this property to "True" and may be also deprecate the existing Python version (after a discussion in the dev list).

Reply:
Yeah sounds pretty comprehensive 👍🏽